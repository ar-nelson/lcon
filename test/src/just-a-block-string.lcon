"""
  This is just a block string.
  The newline should be preserved, but the indents should not.
