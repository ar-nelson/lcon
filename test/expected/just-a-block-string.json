"This is just a block string.\nThe newline should be preserved, but the indents should not."
